gradle.properties

```diff
@@ -67,3 +67,13 @@ NUM_SLAVES=3
 hostname=0.0.0.0
 username=root
 
+#Default config values for system test execution on AWS
+#To override this use -Paws_access_key=<> and so on...
```

Comment:
It is not clear where these values will be set. (e.g. -Paws_access_key=). Can you add a small blurb mentioning that?

Reply:
done.